bazel/tools/check_bazel_version.bzl

```diff
@@ -0,0 +1,30 @@
+"""Check Bazel version against .bazelversion file.
```

Comment:
Let's stick to bazel conventions: please, rename file and use underscore instead of a dash